releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- Fixed a bug introduced in v19.2 that would allow foreign keys to use a unique index on the referenced columns that indexed more columns than were included in the columns used in the FK constraint, which allows potentially violating uniqueness in the referenced columns themselves. [#43793][#43793]
+- Ensure that RESTORE cleanup is run exactly once. [#43933][#43933]
+- A benign error previously logged at the ERROR level is now logged at the INFO level behind a verbosity(2) flag. This error might have been observed as "context canceled: readerCtx in Inbox stream handler". [#44020][#44020]
+- A bug causing lost update transaction anomalies was fixed. [#42969][#42969]
+- Previously, an internal error could occur when a query with an aggregate function MIN or MAX was executed via the vectorized engine when the input column was either INT2 or INT4 type. This is now fixed. [#43985][#43985]
+- CDC is no longer susceptible to a bug where a resolved timestamp might be published before all events that precede it have been published in the presence of a Range merge. [#44035][#44035]
+- `cockroach zip` now emits the `goroutine` file in the proper sub-directory when the corresponding call fails with an error. [#44064][#44064]
+- `cockroach debug zip` is again able to operate correctly and continue to iterate over all nodes if one of the nodes does not deliver its goroutine dumps. It would previously prematurely and incorrectly stop with an incomplete dump; this was a regression introduced in v19.2. [#44064][#44064]
+- The file generated by `cockroach gen haproxy` no longer gets an executable bit. The executable bit was previously placed in error. [#44043][#44043]
+- Fixed internal error of the form "x FK cols, only y cols in index" in some cases when inserting into a table with foreign key references. [#44031][#44031]
+- CockroachDB now ensures the import cleanup is only run once. [#43960][#43960]
+- Converted a panic in `golang.org/x/text/language/tags.go` when using collated strings to an error. [#44103][#44103]
+- SQL mutation statements that target tables with no foreign key relationships now correctly read data as per the state of the database when the statement started execution. This is required for compatibility with PostgreSQL and to ensure deterministic behavior when certain operations are parallelized. Prior to this fix, a statement [could incorrectly operate multiple times](https://en.wikipedia.org/wiki/Halloween_Problem) on data that itself was writing, and potentially never terminate. This fix is limited to tables without FK relationships, and for certain operations on tables with FK relationships; in other cases, the fix is not active and the bug is still present. A full fix will be provided in a later release. [#42862][#42862]
```

Comment:
Should add the text "halloween problem" somewhere in here so people can search for it (it's in the title of the wikipedia link).